ghost/core/core/server/data/migrations/versions/5.53/2023-06-20-10-18-add-collections-table.js

```diff
@@ -0,0 +1,13 @@
+// For information on writing migrations, see https://www.notion.so/ghost/Database-migrations-eb5b78c435d741d2b34a582d57c24253
+
+const {addTable} = require('../../utils');
+
+module.exports = addTable('collections', {
+    id: {type: 'string', maxlength: 24, nullable: false, primary: true},
+    title: {type: 'string', maxlength: 191, nullable: false},
+    slug: {type: 'string', maxlength: 191, nullable: false, unique: true},
+    description: {type: 'string', maxlength: 2000, nullable: true},
+    filter: {type: 'string', maxlength: 2000, nullable: true},
+    type: {type: 'string', maxlength: 50, nullable: false},
+    feature_image: {type: 'string', maxlength: 2000, nullable: true}
+});
```

Comment:
I think collections are going to need meta the same as tags. Ideally we'd farm this out and flatten it but I guess short term it's best to have it inline?

We definitely need it because collections are going to represent index pages and these will want to have meta data.

Reply:
This got missed, I'll try and make time tomorrow to get a second migration prepared for next week for these